modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JaxRSServerCodegen.java

```diff
@@ -167,6 +167,15 @@ public void addOperationToGroup(String tag, String resourcePath, Operation opera
     }
 
     @Override
+    public String toApiName(String name) {
+        if (name.length() == 0) {
+            return "DefaultApi";
+        }
+        name = name.replaceAll("[^a-zA-Z ]+", "_");
```

Comment:
what about numbers? it's probably unusual but that doesn't mean we should remove them!?